apps/odyssey-stats/src/widget/highlights.jsx

```diff
@@ -0,0 +1,71 @@
+import classNames from 'classnames';
+import { useTranslate } from 'i18n-calypso';
+import moment from 'moment';
+import useReferrersQuery from '../hooks/use-referrers-query';
+import useTopPostsQuery from '../hooks/use-top-posts-query';
+
+import './hightlights.scss';
+
+function TopColumn( { items, viewAllUrl, viewAllText, title, className = null } ) {
+	const translate = useTranslate();
+	// TODO: add a placeholder state
```

Comment:
What would this placeholder state do?

Reply:
It is like the loading spinner on the chart, the same as [your mentioned comment](https://github.com/Automattic/wp-calypso/pull/75323#discussion_r1162138268). 😄